### DISCUSSION TITLE
Deployment method inquiry.

### DISCUSSION THREAD

**Discussion by xiangyingchang** [n3d1117/chatgpt-telegram-bot#431] [Q&A]
Hello, I would like to ask how can I deploy this robot to services such as Vercel so that I can use it remotely after shutting down my computer?

**Comment by noriellecruz** [comment]
this is not an issue. please post it in the **discussions** tab.

**Comment by xiangyingchang** [comment]
> this is not an issue. please post it in the **discussions** tab.

ok thx
